nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -1113,6 +1114,9 @@ public void start(final boolean monitor) throws IOException {
             cmdLogger.warn("Failed to delete previous lock file {}; this file should be cleaned up manually", prevLockFile);
         }
 
+        final RuntimeValidatorChecker configChecker = new RuntimeValidatorChecker();
+        configChecker.check();
```

Comment:
This works, but it looks like the `RuntimeValidatorChecker` could be declared once as a static variable and reused.